.changeset/seven-plants-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
+---
+
+Add type to remove button avoiding unintentional form submission
```

Comment:
We should add the component that's affected. Otherwise, you don't know what component is affected by reading the change logs.

```suggestion
[FileUploader] - Add type to remove button avoiding unintentional form submission
```